app/build.gradle.kts

```diff
@@ -69,7 +69,10 @@ android {
         unitTests {
             isIncludeAndroidResources = true
             isReturnDefaultValues = true
+
+
```

Comment:
Question: Does the ktfmt formatter really add these empty lines, or can you remove those lines?

Reply:
I fixed it. Thank you!